en/docs/learn/examples/inbound-endpoint-examples/inbound-endpoint-secured-websocket.md

```diff
@@ -1,36 +1,24 @@
-# Using the Secure WebSocket Inbound Endpoint
+# Using the secure WebSocket inbound endpoint
 
-If you need to read and transform the content of WebSocket frames, the
-information in incoming WebSocket frames is not sufficient because the
-WebSocket protocol does not specify any information about the
-content-type of frames that flow through WebSocket channels. Hence, the
-Micro Integrator supports a WebSocket
-subprotocol extension to determine the content type of WebSocket frames.
+If you need to read and transform the content of WebSocket frames, the information in incoming WebSocket frames is not sufficient because the WebSocket protocol does not specify any information about the content-type of frames that flow through WebSocket channels. Hence, the Micro Integrator supports a WebSocket subprotocol extension to determine the content type of WebSocket frames.
```

Comment:
```suggestion
If you need to read and transform the content of WebSocket frames, the information in incoming WebSocket frames may not be sufficient because the WebSocket protocol does not specify any information about the content-type of frames that flow through WebSocket channels. Hence, the Micro Integrator supports a WebSocket subprotocol extension to determine the content type of WebSocket frames.
```